### ISSUE TITLE
Improve insight into Open Zaak performance under duress

### ISSUE THREAD

**Issue by alextreme** [open-zaak/open-zaak#1760]
### Thema / Theme

Other

### Omschrijving / Description

In order to improve insight into the CPU usage of Open Zaak on our nikki performance tests, we can introduce:
- Django silk
- New Relic APM
- Activating the Postgresql slow query log on nikki

**Comment by sergei-maertens** [comment]
See also docs: https://open-zaak.readthedocs.io/en/stable/development/performance/profiling.html :)

**Comment by PeterVanBragt** [comment]
Let's wait for OZ 1.14 before we go for further analysing

**Comment by JanBrek** [comment]
@joeribekker One thing that I am thinking of which might help with (db) performance is the caching of produced -types. Due to the nature of these, they are not subject to any change afaik. Hence it should not bring any issues to operations and outdated data.
I am sure that the reality is a bit more nuanced but it still might be something additional to look into. This might come in especially handy with the expansion functionality so that [part of] the expansions dont take too much extra resources to be included.

**Comment by joeribekker** [comment]
@JanBrek yes, I agree. We investigated this a while ago and should be relatively easy. We focused on the Zaken and Documenten API because those were the main issue. I don't know if the Catalog API is worth optimizing.

**Comment by joeribekker** [comment]
I think this is picked up under #1926
